packages/jupyter-chat/src/widgets/chat-widget.tsx

```diff
@@ -321,6 +328,41 @@ export class ChatWidget extends ReactWidget {
     }
   }
 
+  /**
+   * Process dropped tabBar files
+   */
+  private _processTabDrop(event: Drag.Event) {
+    const factory = event.mimeData.getData(TABBAR_FILE_MIME) as () => any;
+    if (!factory) {
+      console.warn('No factory in drag event');
+      return;
+    }
+
+    const widget = factory();
+    if (!widget) {
+      console.warn('Factory did not return a widget');
+      return;
+    }
+
+    const path =
+      typeof widget.context?.path === 'string' ? widget.context.path : null;
+    if (!path) {
+      console.warn('Widget has no path');
+      return;
+    }
```

Comment:
```suggestion
    if (!widget || !(widget instanceof DocumentWidget) {
      console.warn('No file associated to the element');
      return;
    }

    const path = widget.context.path;
    if (!path) {
      console.warn('Widget has no path');
      return;
    }
```

Let's check for the Widget type, if it is not a `DocumentWidget`, no need to carry on.

Reply:
Done 🎉